### PR TITLE
fix(#365): Added SPAship definition & Footer info to SPAship login page.

### LINKED ISSUE #365: [Bug]: SPAship definition and Footer info missing in Login Page
## Describe the bug:
As per the SPAship manager designs the login page was supposed to have the SPAship definition and the footer info included. However, that is currently missing. The missing sections need to be added to the login page.

### PR DESCRIPTION
**Pull request by sayak-sarkar**
## Closes / Fixes / Resolves
Resolves #365 

## Explain the feature/fix
Added SPAship definition and Footer info to the `/login` page.

## Does this PR introduce a breaking change
No

## Screenshot(s)
image

### Ready-for-merge Checklist
- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Did tests pass?

### CODE REVIEW

File: packages/manager/src/components/user/LoginPage.tsx [hunk 1]
```diff
@@ -52,22 +77,47 @@ export default () => {
...
+            <Button isBlock variant="primary" onClick={onClickLogin} className="spaship_btn">
+              Red Hat SSO
+            </Button>
+          </CardBody>
+        </Card>
+      </PageSection>
+      <PageSection
+        className={css(styles.footer)}
+        variant={PageSectionVariants.darker}
+        style={{ height: "6em" }}
```
Comment (kunyan) [inline packages/manager/src/components/user/LoginPage.tsx:104]: Why not move this into the top style definition ?
```suggestion

```

File: packages/manager/src/components/user/LoginPage.tsx [hunk 2]
```diff
@@ -52,22 +77,47 @@ export default () => {
...
+        isFilled={false}
+      >
+        Brought to you by the{" "}
+        <a href="https://github.com/spaship/spaship/graphs/contributors" target="_">
```
Comment (kunyan) [inline packages/manager/src/components/user/LoginPage.tsx:108]: The `target` should be `_blank`,  the `_` is invalid in target attribute .

and `rel="noreferrer noopener"` is recommend if you are use `target`
ref: [<a>: The Anchor element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a)

```suggestion
        <a href="https://github.com/spaship/spaship/graphs/contributors" target="_blank" rel="noreferrer noopener">
```

File: packages/manager/src/components/user/LoginPage.tsx [hunk 3]
```diff
@@ -52,22 +77,47 @@ export default () => {
...
+          Wizards <OptimizeIcon />
+        </a>{" "}
+        of the{" "}
+        <a href="https://github.com/spaship/spaship" target="_">
```
Comment (kunyan) [inline packages/manager/src/components/user/LoginPage.tsx:112]:
```suggestion
        <a href="https://github.com/spaship/spaship" target="_blank" rel="noreferrer noopener">
```

File: packages/manager/src/components/user/LoginPage.tsx [hunk 4]
```diff
@@ -52,22 +77,47 @@ export default () => {
...
+          SPAship
+        </a>{" "}
+        project. <br /> Code licensed under the{" "}
+        <a href="https://github.com/spaship/spaship/blob/master/LICENSE" target="_">
```
Comment (kunyan) [inline packages/manager/src/components/user/LoginPage.tsx:116]:
```suggestion
        <a href="https://github.com/spaship/spaship/blob/master/LICENSE" target="_blank" rel="noreferrer noopener">
```

File: packages/manager/src/components/user/LoginPage.tsx [hunk 5]
```diff
@@ -8,32 +8,57 @@ import {
...
-    zIndex: 0
+    zIndex: 0,
   },
   button: {
-    backgroundColor: "#fed402"
-  }
+    backgroundColor: "#fed402",
+  },
+  footer: {
+    fontSize: "0.6rem",
```
Comment (kunyan) [inline packages/manager/src/components/user/LoginPage.tsx:60]:
```suggestion
    fontSize: "0.6rem",
    height: "6em"
```

File: packages/manager/src/components/user/LoginPage.tsx (hunk 1 above)
Comment (sayak-sarkar) [reply packages/manager/src/components/user/LoginPage.tsx:104]: Will do!

File: packages/manager/src/components/user/LoginPage.tsx (hunk 2 above)
Comment (sayak-sarkar) [reply packages/manager/src/components/user/LoginPage.tsx:108]: Done.

**Pull request merged**